City-of-Helsinki/drupal-helfi-platform-config
# Uhf 91 create test content

To test:

- use the oneliner to install HELfi platform
- switch config to this branch with `composer require drupal/helfi_platform_config:dev-UHF-91_create_test_content`
- `composer install` tai `make new`
- `make shell`
- `drush en helfi_example_content -y`
- the module should install and afterwards node listing should include a node in admin panel
